app/gui/src/controller/breadcrumbs.rs

```diff
@@ -0,0 +1,44 @@
+//! A module containing a controller for the breadcrumbs panel of the component browser.
+
+use crate::prelude::*;
+
+use crate::controller::searcher::component;
+
+
+
+// ===================
+// === Breadcrumbs ===
+// ===================
+
+/// Breadcrumbs controller.
+///
+/// This object is responsible for interactions between the breadcrumbs and the searcher controller.
```

Comment:
I think the presenter should be responsible for "interactions", not controller. I'd rephrase it to something like "a controller keeping the path of entered modules in Searcher and providing API for breadcrumbs panel."